aws/s3/ingestBills.go

```diff
@@ -89,6 +90,14 @@ func UpdateDueReports(ctx context.Context, tx *sql.Tx) ([]ReportUpdateConclusion
 			}
 			aas[br.AwsAccountId] = aa
 		}
+		err = es.CleanCurrentMonthBillByBillRepositoryId(ctx, aa.UserId, br.Id)
```

Comment:
You've cleaned when we call ingest-due task but you've forgotten the ingest task. Ingest task is the only one used in production.